script.skin.helper.colorpicker/addon.xml

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+	<addon id="script.skin.helper.colorpicker" name="Skin Helper Service ColorPicker" version="1.0.0" provider-name="marcelveldt">
+	<requires>
+		<import addon="xbmc.python" version="2.13.0"/>
+		<import addon="xbmc.addon" version="12.0.0"/>
+		<import addon="script.module.pil" version="1.1.7"/>
```

Comment:
Maybe you're missing it as the diff might be to large for Github to display properly...

https://github.com/marcelveldt/script.skin.helper.colorpicker/blob/master/resources/lib/ColorPicker.py#L13

https://github.com/marcelveldt/script.skin.helper.colorpicker/blob/master/resources/lib/ColorPicker.py#L73

Offcourse if you have a great idea how to avoid the creating of the images for each color, I'd be very happy to drop PIL immediately :-)
What I already tried was setting a listitem property to the color hex string but I can't use that as colordiffuse within the itemtemplate